docs/kb/semgrep-supply-chain/sss-python-lockfiles.md

```diff
@@ -4,48 +4,60 @@ tags:
   - Semgrep Supply Chain
   - Python
   - Knowledge Base
-  - Lock files
+  - Lockfiles
```

Comment:
"Lockfiles" is our preferred term that is used in the existing docs. (You can use Vale to help remember).